src/components/Text.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+export default function Text({ 
+  text = "",
+  italic = false, 
+  bold = false, 
+  underline = false, 
+  size = 'text-base', // uses tailwind utility class to set 1rem
+  color = 'text-black', 
+}) { 
+    const classNames = 
+    (italic? 'italic ' : '') + 
+    (bold? 'font-bold ' : '') + 
+    (underline? 'underline ' : '') + 
+    size + ' ' + 
+    color;
```

Comment:
(nit) Again from above, slight inconsistency with giving spaces between two class names vs having the spacing be added with the classname